FV3/gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -4629,14 +4630,15 @@ subroutine GFS_physics_driver                                 &
             call set_state("specific_humidity_after_gscond", Stateout%gq0(1:im, 1:levs, 1))
             call set_state("air_temperature_after_gscond", Stateout%gt0(1:im, 1:levs))
             call set_state("cloud_water_mixing_ratio_after_gscond", Stateout%gq0(1:im, 1:levs, ntcw))
+            
+            ! if gscond model not specified, does nothing and diags should be equivalent w/ physics
```

Comment:
If it can be straightforwardly diagnosed (no worries if not), would it make sense to add some sort of log statement in this scenario?

Reply:
We do have log statements in the python hook for the emulators that signify no models are being used. ([link](https://github.com/ai2cm/fv3net/blob/9fc23fc2401b09066842b6c1e2856e36bfc5828f/external/emulation/emulation/config.py#L214))